Lib/inspect.py

```diff
@@ -2171,7 +2171,8 @@ def _signature_from_callable(obj, *,
 
     # Was this function wrapped by a decorator?
     if follow_wrapper_chains:
-        obj = unwrap(obj, stop=(lambda f: hasattr(f, "__signature__")))
```

Comment:
Need to add a comment here explaining the "or" in unwrap lambda.

Reply:
If you've missed this change — I've added this in commit below. Should this branch be resolved to continue review of this PR?